fdbserver/TenantBalancer.actor.cpp

```diff
@@ -1008,7 +1021,7 @@ ACTOR Future<Void> finishSourceMovement(TenantBalancer* self, FinishSourceMoveme
 
 		// Check that the DR is ready to switch
 		if (targetMovementInfo.databaseBackupStatus == "has errored") {
-			record.movementState = MovementState::ERROR;
+			record.setMovementError("Database backup has error.");
```

Comment:
```suggestion
			record.setMovementError("The DR copying the data has failed.");
```

Do we have an error string from DR? If so, we should include that too.

Reply:
we only record secondsBehind and backupStatus from DR, without error string for now. And I'm not sure if there is such error message string. Do you know if there is any error message string in DR? I'll check later if you're not sure.